.browserslistrc

```diff
@@ -1,10 +1 @@
-# Browsers that we support. To see current list: `npx browserslist`
-
-last 2 major versions
-last 2 years
-> 1%
-not dead
-not op_mini all
-not op_mob > 0
-not ie > 0
-not ie_mob > 0
+extends @goodhood/browserslist-config-nebenan
```

Comment:
maybe publish as just `browserslist-config-nebenan` and move in a separate repo? this repo holds react ui packages

Reply:
Doesn't really matter imo. Doesn't complicate setup in any way

if you only want to have react stuff in here, then you chose a very generic scope name